cluster/config-defaults.yaml

```diff
@@ -168,6 +168,12 @@ teapot_admission_controller_deployment_default_max_surge: "5%"
 teapot_admission_controller_deployment_default_max_unavailable: "1"
 teapot_admission_controller_inject_aws_waiter: "true"
 
+teapot_admission_controller_check_daemonset_resources: "true"
```

Comment:
Do you want to immediately turn it on?

Reply:
We'd see how it work in `dev`, but yeah, we could also be on the safe side.